ansible/workers.yml

```diff
@@ -12,6 +12,11 @@
       name: jenkins
       comment: "Jenkins user"
 
+  - name: Install prerequisites for authorized keys addition
```

Comment:
we need selinux for authorized keys??

Reply:
Believe it or not, the following task was failing and complained about missing selinux Python bindings. I assume that it may have been related to something like https://www.pyrosoft.co.uk/blog/2013/01/12/setting-up-ssh-authorized_keys-with-selinux-enabled/ , but I am not sure either.